pkg/scan/process_image.go

```diff
@@ -873,3 +883,16 @@ func (s *Scanner) ExtractAndScanFromTar(tarFolder string) (*ImageExtractionResul
 	}
 	return &ImageExtractionResult{ImageId: imageScan.imageId, IOCs: IOCs}, nil
 }
+
+func CheckScanStatus(s *Scanner) error {
+	if s != nil && s.ReportStatus == true {
+		if s.Aborted == true {
```

Comment:
`Aborted` & `ReportStatus` should be `atomic.Bool` since they might be modified within another thread  
(Same in `SecretScanner` actually)